File: Civitas-Java/Civitas/src/main/java/civitas/TestP1.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package civitas;

import java.util.ArrayList;
import java.io.*;

/**
 *
 * @author elena
 */
public class TestP1 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {  
        ArrayList<String> nombres = new ArrayList();
        nombres.add("Elena");
        nombres.add("Pepe");
        nombres.add("Juan");
        nombres.add("Laura");
        
        CivitasJuego juego = new CivitasJuego(nombres);
        
        System.out.print(juego.infoJugadorTexto());
        System.out.print("\n");
        //System.out.print(juego.getCasillaActual().toString());
        /*System.out.print("\n\n");
        System.out.print(juego.finalDelJuego());        
        System.out.print("\n");*/
        //juego.pasarTurno();
        //System.out.print(juego.vender(1));
        
    }
    
}

File: Civitas-Ruby/pkg/Civitas-Ruby-0.0.1/lib/test_p1.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

require_relative "dado"
require_relative "tipo_casilla"
require_relative "tipo_sorpresa"
require_relative "operaciones_juego"
require_relative "estados_juego"
require_relative "diario"
require_relative "casilla"
require_relative "tablero"
require_relative "civitas_juego"
require_relative "gestor_estados"
require_relative "mazo_sorpresas"
require_relative "sorpresa"

module Civitas
  class TestP1
        
    def self.main
      @nombres = Array.new
      @nombres << 'Elena'
      @nombres << 'Pepe'
      @nombres << 'Laura'
      @nombres << 'Juan'
      
      @juego = CivitasJuego.new(@nombres)
      
      #puts 'hola'
      puts @juego.infojugadortexto
    end
  end
  
  TestP1.main
end
